src/typography/__tests__/Text.test.js

```diff
@@ -70,3 +70,16 @@ describe('Sizing', () => {
     `)
   })
 })
+
+describe('Props', () => {
+  it('should forward `className` prop', () => {
+    const MY_CLASS = 'my-custom-class'
+    const component = (
+      <Text data-testid="text" className={`${MY_CLASS}`}>
```

Comment:
Nitpick: String interpolation isn't necessary here. Also, it shouldn't _really_ matter, but any thoughts on randomizing the class name similar to the same test for `<Code />`?

Reply:
Ah, missed that. Good point, updated in 2897794